dashboard/models.py

```diff
@@ -344,6 +346,7 @@ def add_timepoint(self, timepoint):
             e.message = "Failed to add timepoint {}. Reason: {}".format(
```

Comment:
Line 340, `FlushError` function not imported.

Reply:
Ah! Whoops. Another good catch, thank you!